src/main/java/io/dashboardhub/pipelinedashboard/controller/AccountController.java

```diff
@@ -16,11 +16,11 @@
 public class AccountController {
 
     @Autowired
-    private UserService userService;
+    private UserServiceImpl userServiceImpl;
```

Comment:
Should be coding against the Interface here and Autowired will pick the correct (and only) implementation:

```
@Autowired
private UserService userService;
```